src/docs/development/ui/assets-and-images.md

```diff
@@ -340,6 +340,13 @@ NSString* path = [[NSBundle mainBundle] pathForResource:key ofType:nil];
 For a more complete example, see the implementation of the
 Flutter [video_player plugin][].
 
+### Loading iOS Images in Flutter
+
+If you are implementing Flutter as
+[Add-to-app](/docs/development/add-to-app/ios/), you may have images hosted in
+iOS which you want to use in Flutter.  For accomplishing that there is a plugin
```

Comment:
Rewrite to avoid passive voice and 
To accomplish that, check out the [ios_platform_images][] plugin on pub.dev.

go/dart-terms: https://docs.google.com/document/d/1vClxUPrWU1fsRq5Ws_KSz6cZHJ2QxkCCG436HJNNGh4/edit#heading=h.6tus0dk5ek5u

Reply:
done